Dockerfile

```diff
@@ -6,6 +6,10 @@ WORKDIR /application
 
 ENV DEBIAN_FRONTEND noninteractive
 
+RUN apt-get install -y ca-certificates
+COPY ./docker/ssl.crt /usr/local/share/ca-certificates/ssl.crt
+RUN update-ca-certificates
```

Comment:
What do we need those certs for?

Reply:
Removed. It shouldn't be there. Needed them to overcome proxy issues